services/galley/src/Galley/API/Internal.hs

```diff
@@ -646,8 +630,13 @@ deleteFederationDomainLocalUserFromRemoteConversation dom = do
       localDomain = env ^. Galley.App.options . optSettings . setFederationDomain
   -- Process each user.
   for_ (Map.toList rCnvMap) $ \(cnv, lUsers) -> do
-    -- All errors, either exceptions or Either e, get thrown into IO
-    mapError @NoChanges (const (InternalErrorWithDescription "No Changes: Could not remove a local member from a remote conversation.")) $ do
+    let catchBaddies =
+          logAndIgnoreErrors @NoChanges
+            ( const "NoChanges: Could not remove a local member from a remote conversation."
+            -- `NoChanges` doesn't contain too many details, so no point in showing it here.
+            )
+            "Federation domain removal"
+    catchBaddies $ do
```

Comment:
mostly to minimize whitespace changes, but i also like the name.  :)

Reply:
Both are nice reasons! The name is short and to the point.